### ISSUE TITLE
Does it support sonarqube 7.0

### ISSUE THREAD

**Issue by toanstt** [stevespringett/dependency-check-sonar-plugin#52]

**Comment by pethers** [comment]
Yes it works, run it myself on https://www.hack23.com/sonar/component_measures?id=com.hack23.cia%3Acia-all . No vulnerabilities, but have resolved them.

**Comment by stevespringett** [comment]
@pethers thanks for the info. I haven't tested 7 yet. I still need to look at their list of proposed changes for 7LTS to see what's coming.

**Comment by toanstt** [comment]
Thanks @pethers

**Comment by toanstt** [comment]
I see it is not stable. For some first scans it worked but now the scanning background on SonarQube always fail with the same error "java.lang.IllegalArgumentException: Plugin [dependencycheck] does not exist"

**Comment by ysnoussi** [comment]
How can one install this plugin ? i dropped the jar in /extensions but I don't see anything/. is there any setup to do ?

**Comment by Reamer** [comment]
Please drop the jar in .../extensions/plugins/ folder and restart sonarqube. No setup needed.

**Comment by ysnoussi** [comment]
Thanks for your reply, this is exactly what I did I did a scan with dependency-check (node based project) run sonar than I was to see the vulnerabilities in the html output but none was displayed owasp-depenedency-ceck under measures

**Comment by Reamer** [comment]
I think you have the same problem as described in #82.
It would be nice, if you build the plugin be yourself based on current master branch (sonarqube 7.3 and above is required) or on branch SonarQube_6.x (sonarqube 6.7.5 and above is required), install this jar in your sonarqube and test again.
